CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ enable_language(C)
 
 # ----- CXX
 enable_language(CXX)
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
This should actually be updated to use `target_compile_features(... PUBLIC cxx_std_17)`. It should be possible to compile this with C++20 as well.

Reply:
Good point!

Reply:
I've just updated `dlaf.prop` target (I missed that) and I removed the line above since it should be enough the former one.